commands/scan/buildscan.go

```diff
@@ -145,19 +145,21 @@ func (bsc *BuildScanCommand) runBuildScanAndPrintResults(xrayManager *xray.XrayS
 	log.Info("The scan data is available at: " + buildScanResults.MoreDetailsUrl)
 	isFailBuildResponse = buildScanResults.FailBuild
 
-	cmdResults := results.NewCommandResults(utils.Build).SetXrayVersion(xrayVersion)
+	cmdResults := results.NewCommandResults(utils.Build).
+		SetXrayVersion(xrayVersion).
+		SetResultsContext(results.ResultContext{ProjectKey: params.Project, IncludeVulnerabilities: bsc.includeVulnerabilities})
```

Comment:
Question: for buildScan we can only receive Project key? what about watches directly?

Reply:
only project
![image](https://github.com/user-attachments/assets/3110d484-e063-4da4-abd6-d4dbb19bb51e)